Zend/zend_exceptions.stub.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+interface Throwable
+{
+    /** @return string */
+    function getMessage();
+
+    /** @return int */
+    function getCode();
+
+    /** @return string */
+    function getFile();
+
+    /** @return int */
+    function getLine();
+
+    /** @return array */
+    function getTrace();
+
+    /** @return ?Throwable */
+    function getPrevious();
+
+    /** @return string */
+    function getTraceAsString();
+
+    /** @return string */
+    function __toString();
+}
+
+class Exception implements Throwable
+{
+    final private function __clone() {}
+
+    function __construct(string $message = UNKNOWN, int $code = 0, ?Throwable $previous = null) {}
+
+    function __wakeup() {}
+
+    /** @return string */
+    final function getMessage() {}
```

Comment:
As these are final methods, it should be fine to add `: string`. However, because these work by reading properties, this might not be guaranteed right now, so we should probably hold off on it...

Reply:
ACK. We can revisit this later.